tests/testsuite/precise_pre_release.rs

```diff
@@ -0,0 +1,61 @@
+//! Tests for selecting pre-release versions with `update --precise`.
+
+use cargo_test_support::project;
+
+#[cargo_test]
+pub fn requires_nightly_cargo() {
```

Comment:
nit: `pub` is unnecessary.

```suggestion
fn requires_nightly_cargo() {
```